examples/BLUEPRINT/nova/curved_picture_frame_example.py

```diff
@@ -38,23 +38,23 @@
 params = [
     ["R_0", "Major radius", 3.639, "m", None, "Input"],
     ["B_0", "Toroidal field at R_0", 2.0, "T", None, "Input"],
-    ["n_TF", "Number of TF coils", 12, "N/A", None, "Input"],
+    ["n_TF", "Number of TF coils", 12, "dimensionless", None, "Input"],
     ["tk_tf_nose", "TF coil inboard nose thickness", 0.17, "m", None, "Input"],
     ['tk_tf_side', 'TF coil inboard case minimum side wall thickness', 0.02, 'm', None, 'Input'],
     ["tk_tf_wp", "TF coil winding pack thickness", 0.569, "m", None, "PROCESS"],
     ["tk_tf_ins", "TF coil ground insulation thickness", 0.008, "m", None, "Input"],
     ["tk_tf_insgap", "TF coil WP insertion gap", 1.0E-7, "m", "Backfilled with epoxy resin (impregnation)", "Input"],
     ["r_tf_in", "Inboard radius of the TF coil inboard leg", 0.148, "m", None, "PROCESS"],
-    ["TF_ripple_limit", "Ripple limit constraint", 0.65, "%", None, "Input"],
+    ["TF_ripple_limit", "Ripple limit constraint", 0.65, "dimensionless", None, "Input"],
```

Comment:
0.0065 (note that we treat the number in % in the actual solvers)

Reply:
ok good to know